oms-status/docker/cachet/.env.example

```diff
@@ -43,3 +43,8 @@ NEXMO_SMS_FROM=Cachet
 
 TRUSTED_PROXIES=
 
+SLACK_WEBHOOK_URL=
```

Comment:
do we have the provisioning for this file?

Reply:
we could, good idea